xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -115,7 +117,21 @@ bool CRendererDXVA::Configure(const VideoPicture& picture, float fps, unsigned o
         m_processor->ListSupportedConversions(dxgi_format, dest_format, picture);
 
       if (m_processor->IsFormatConversionSupported(dxgi_format, dest_format, picture))
+      {
+        const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+
+        if (!settings)
+          return true;
+
+        if (settings->GetBool(CSettings::SETTING_VIDEOPLAYER_USESUPERRESOLUTION) &&
+            DXVA::CProcessorHD::IsSuperResolutionSuitable(picture) &&
+            DX::Windowing()->SupportsVideoSuperResolution())
```

Comment:
Probably change order here is sufficient  

move `DX::Windowing()->SupportsVideoSuperResolution() ` first

Reply:
> move `DX::Windowing()->SupportsVideoSuperResolution() ` first

Yes that takes care of it.